tests/zero-downtime-scaling/results/1.2.0/1.2.0.md

```diff
@@ -0,0 +1,578 @@
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     1.50ms  773.48us  34.14ms   92.03%
+    Req/Sec   687.59    106.56     0.97k    70.74%
+  Latency Distribution
+     50%    1.37ms
+     75%    1.67ms
+     90%    2.06ms
+     99%    4.20ms
+  351750 requests in 4.28m, 125.39MB read
+  Socket errors: connect 0, read 16, write 0, timeout 0
+Requests/sec:   1368.69
+Transfer/sec:    499.62KB
+```
+
+HTTP Graph:
+
+![gradual-up-http.png](10-node/gradual-up-http.png)
+
+
+HTTPS wrk output
+
+```text
+wrk -t2 -c2 -d5m --latency --timeout 2s  https://cafe.example.com/tea
+Running 5m test @ https://cafe.example.com/tea
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     1.60ms  769.01us  36.29ms   92.22%
+    Req/Sec   641.09     97.84     0.96k    70.71%
+  Latency Distribution
+     50%    1.47ms
+     75%    1.77ms
+     90%    2.16ms
+     99%    4.27ms
+  328759 requests in 4.29m, 114.88MB read
+  Socket errors: connect 0, read 16, write 0, timeout 0
+Requests/sec:   1276.21
+Transfer/sec:    456.64KB
+```
+
+HTTPS Graph:
+
+![gradual-up-https.png](10-node/gradual-up-https.png)
+
+Logs:
+
+- NGF error logs: none
+- NGINX error/warn logs: none
+- NGINX access logs
+  - 842986 [200 Oks]
+  - One 499 `INFO 2024-03-19T21:28:15.266352183Z [resource.labels.containerName: nginx] 10.150.0.27 - - [19/Mar/2024:21:28:15 +0000] "GET /tea HTTP/1.1" 499 0 "-" "-"`
+
+
+### Scale Down Gradually
+
+HTTP wrk output
+
+```text
+wrk -t2 -c2 -d20m --latency --timeout 2s  http://cafe.example.com/coffee
+Running 20m test @ http://cafe.example.com/coffee
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     8.99ms   71.48ms   1.07s    98.65%
+    Req/Sec   704.32     82.44     0.95k    74.78%
+  Latency Distribution
+     50%    1.35ms
+     75%    1.63ms
+     90%    2.03ms
+     99%  326.29ms
+  1603461 requests in 19.81m, 570.38MB read
+  Socket errors: connect 0, read 21, write 0, timeout 10
+Requests/sec:   1349.02
+Transfer/sec:    491.38KB
+```
+
+HTTP Graph
+
+![gradual-down-http.png](10-node/gradual-down-http.png)
+
+
+HTPPS wrk output
+
+```text
+wrk -t2 -c2 -d20m --latency --timeout 2s  https://cafe.example.com/tea
+Running 20m test @ https://cafe.example.com/tea
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     7.13ms   61.54ms   1.07s    98.98%
+    Req/Sec   661.77     77.50     0.90k    75.68%
+  Latency Distribution
+     50%    1.43ms
+     75%    1.73ms
+     90%    2.13ms
+     99%   85.50ms
+  1500047 requests in 19.75m, 523.58MB read
+  Socket errors: connect 0, read 27, write 0, timeout 12
+Requests/sec:   1265.79
+Transfer/sec:    452.41KB
+```
+
+HTTPS Graph
+
+![gradual-down-https.png](10-node/gradual-down-https.png)
+
+Logs:
+
+- NGF error logs: 25
+
+All or most error were related to failure collecting telemetry . The below error occurred 25 times.
```

Comment:
> All or most error were related to failure collecting telemetry 

do we have any other errors not related to telemetry? Those could be other bugs

Reply:
No we don't. I attached the log files to [story](https://github.com/nginxinc/nginx-gateway-fabric/issues/1738).

Reply:
Could we reword that sentence? For example, "All errors were related to failure collecting telemetry". Otherwise, it indicates that there were other errors, which isn't the case.